i18n/create_msg.js

```diff
@@ -35,25 +38,32 @@ const CATBLOCK_MSGS = `
 Blockly.CatblocksMsgs = {};
 Blockly.CatblocksMsgs.locales = {};
 
-Blockly.CatblocksMsgs.currentLocale_ = 'en';
+Blockly.CatblocksMsgs.currentLocale_ = '${DEFAULT_LANGUAGE}';
 
 Blockly.CatblocksMsgs.hasLocale = function(locale) {
   return Object.keys(Blockly.CatblocksMsgs.locales).includes(locale);
 };
 
 Blockly.CatblocksMsgs.setLocale = function(locale) {
   if (Blockly.CatblocksMsgs.hasLocale(locale)) {
-    Blockly.CatblocksMsgs.currentLocale_ = locale;
-    Blockly.Msg = Object.assign({}, Blockly.Msg, Blockly.CatblocksMsgs.locales[locale]);
+    if (Object.keys(Blockly.CatblocksMsgs.locales[locale]).length === 1) {
+      return Blockly.CatblocksMsgs.loadNewLocale(locale).then(() => {
+        Blockly.CatblocksMsgs.currentLocale_ = locale;
+        Blockly.Msg = Object.assign({}, Blockly.Msg, Blockly.CatblocksMsgs.locales[locale]);
+      });
```

Comment:
check if locale already loaded. if not, then load from json file (in loadNewLocale() function).